WookieeMatt/Essence20
# Megaform trait items; Displaying MFZ combiners

In this change
- Megaform Traits are now items that can be added to Zords
- Improving display of Zord on MFZ sheet
- New header string
- Removing some duplicate strings

Testing
- TEMPLATE CHANGE, so restart Foundry
- Make some Zords, add some Megaform Traits
- Make a MFZ
- Drop the Zords onto the MFZ and they should appear in their section
- Deleting should work

Things to note:
- If you modify the Zord you will have to reopen the MFZ to see the changes take effect. I think this is reasonable?
- We can figure out a nicer layout/format later
